processstarter/src/main/native/linux/main.cpp

```diff
@@ -85,3 +102,22 @@ int main(int argc, char* argv[]) {
   struct pollfd pfd = {childPid, POLLIN, 0};
   return poll(&pfd, 1, 3000);
 }
+
+int StartExeTool(std::filesystem::__cxx11::path& exePath) {
```

Comment:
Why is the inline namespace `__cxx11` specified here?

Reply:
Because the VS Code refactoring tool added it. Removed.